src/scripts/pokemons/PokemonList.ts

```diff
@@ -20817,6 +20817,17 @@ const pokemonMap: any = new Proxy(pokemonList, {
                     const random = Math.floor(Math.random() * (max ? max : pokemon.length) + min);
                     return pokemon[random];
                 };
+            case 'randomRegion':
+                return (_max = GameConstants.Region.kanto, _min = GameConstants.Region.kanto) => {
+                    // minimum 0 (Kanto)
+                    const min = Math.max(GameConstants.Region.kanto, Math.min(_min, _max));
+                    const max = Math.max(GameConstants.Region.kanto, _min, _max);
+                    const filteredPokemon: PokemonListData[] = pokemon.filter(p => p.id > 0 && (p as PokemonListData).nativeRegion >= min && (p as PokemonListData).nativeRegion <= max);
+                    const random = Math.floor(Math.random() * filteredPokemon.length);
+                    const poke: PokemonListData = filteredPokemon[random];
+                    pokemon.find(p => p.id == 0);
```

Comment:
Did you want to do anything with the pokemon you are looking for here?

Reply:
🤦 Was supposed to return that if no Pokémon found 😅

Reply:
Should be fixed now 👍